lib/path.js

```diff
@@ -566,7 +566,7 @@ posix.extname = function(path) {
 posix.format = function(pathObject) {
   if (pathObject === null || typeof pathObject !== 'object') {
     throw new TypeError(
-        "Parameter 'pathObject' must be an object, not " + typeof pathObject
+        'Parameter "pathObject" must be an object, not ' + typeof pathObject
```

Comment:
@micnic want to do a second pass if there are more errors that were missed, like this one?